index.php

```diff
@@ -0,0 +1,131 @@
+<?php
+require 'apicall.php';
+/**
+ * Class fetch to fetch the data from API.
+ * 
+ * Class for fetching title,service,image,alias and icon 
+ */
+class fetch
+{
```

Comment:
Classes should be declared in this format. Make this change everywhere.
```suggestion
class fetch {
```